packages/css/src/components/text-area/README.md

```diff
@@ -0,0 +1,21 @@
+# TextArea
+
+A form field that allows the user to input text over multiple lines.
+
+## Guidelines
+
+- Use a TextArea when users need to enter more than 1 sentence of text, such as a comment or description.
+- The height of the TextArea should be appropriate for the information to be entered.
+  - Use cols and rows attributes to set the width and height of the TextArea.
```

Comment:
The `cols` attribute doesn't really work with our current implementation. We give `textarea` `width: 100%`, which overrides `cols`. What do we want to use, `cols` or `width: 100%`?

`cols` are easier to customise, `width: 100%` works better with `Grid.Cell`

Reply:
I've set the width to auto whenever cols are used.

Reply:
Nice, we can use both then. I would add `max-width: 100%` as well (same for resizing), so you can never break out of the container

Reply:
Added max-width